keep-ui/app/workflows/builder/builder-modal.tsx

```diff
@@ -11,13 +11,15 @@ import { downloadFileFromString } from "./utils";
 interface Props {
   closeModal: () => void;
   compiledAlert: Alert | string | null;
-  id?:string
+  id?: string;
+  hideClose?: boolean;
 }
 
 export default function BuilderModalContent({
   closeModal,
   compiledAlert,
   id,
+  hideClose,
```

Comment:
Let's use cleaner name 
```suggestion
  hideCloseButton,
```